fe/fe-core/src/test/resources/sql/tpchcost/q20.sql

```diff
@@ -322,6 +322,7 @@ actualRows=0, avgRowSize=24.0
 cardinality: 86732673
 probe runtime filters:
 - filter_id = 1, probe_expr = (32: L_PARTKEY)
+- filter_id = 2, probe_expr = (33: L_SUPPKEY)
```

Comment:
In TPC-H 1T or 10T, the performance has any improvement?

Reply:
> > In tpcds-100g, this feature can optimize queries below:
> 
> What's the tpcds-1T and tpcds-10T behavior?

I have not find a very well case in TPCH/TPCDS 1TB/10TB(still work on this). 

Can we merge this first (multi columns global runtime filter is not set by default)? I will add more tests later?